<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Proyecto Emergiendo con el Sol</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Datos actuales</string>
    </property>
    <widget class="QLabel" name="LabelState">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Conexión: </string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelState_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Intervalo de actualización: </string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelState_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Última actualización: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lconnectionInfo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>user@host:port</string>
     </property>
    </widget>
    <widget class="QLabel" name="lrefreshTime">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1 min</string>
     </property>
    </widget>
    <widget class="QLabel" name="llastRefresh">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>07/06/2015</string>
     </property>
    </widget>
    <widget class="QLabel" name="lastRefreshHour">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>22:20:56</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelState_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lState">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuración</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Directorio:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EDir">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>C:\Users\Linkku\Documents\ClientUser\release\workdir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuario:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EUser">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>user0002</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EHost">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>150.214.174.391</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Puerto:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EPort">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10022</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Intervalo actualización (segundos):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ERefresh">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>60</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de ficheros:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EFileType">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>csv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aceptar cambios</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionModify">
   <property name="text">
    <string>modify</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
